src/views/CenterPaneView.tsx

```diff
@@ -21,18 +21,25 @@ export function CenterPaneView({ file, ...props }: CenterPaneViewProps) {
         const textContent = new TextDecoder('utf-8').decode(content);
         setContent(textContent);
         setLoading(false);
-      })()
+      })();
```

Comment:
Unrelated to this PR, but it's weird we have been able to push the code without the semi-colon. Are the pre-push hooks working properly?

Reply:
Done.